checkov/common/parsers/yaml/parser.py

```diff
@@ -0,0 +1,48 @@
+import logging
+
+from yaml import YAMLError
+
+import checkov.common.parsers.yaml.loader as loader
+
+logger = logging.getLogger(__name__)
+
+
+def parse(filename):
+    template = None
+    template_lines = None
+    try:
+        if filename.endswith(".yaml") or filename.endswith(".yml"):
+            (template, template_lines) = loader.load(filename)
+
+        if template:
+            if isinstance(template, list):
+                for t in template:
+                    if t and isinstance(t, dict):
+                        return t, template_lines
+                    if t and isinstance(t, list):
+                        return t, template_lines
```

Comment:
There is a case that t is not dict or list? if so we can just return the first element
Also what about the content in the yaml that we do not return? maybe the parse needs to return list of parsed yamls

Reply:
changed the syntax. honestly, i don't remember why I did that.